Formula/ccache.rb

```diff
@@ -28,8 +28,15 @@ class Ccache < Formula
   fails_with gcc: "5"
 
   def install
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DENABLE_IPO=TRUE"
+    system "cmake", "--build", "build"
+
+    # Homebrew compiler shim actively prevents ccache usage (see caveats), which will break the test suite
```

Comment:
Let's add a comment here explaining why we run `ctest`, in case someone's tempted to remove it in the future.